src/components/lobby/LobbyAwaitGame.tsx

```diff
@@ -0,0 +1,222 @@
+// tslint:disable:no-console
+import * as React from 'react';
+import styled from 'styled-components';
+import BGImage from '../../dist/images/chess-bg.jpg';
+import {CreatedGames} from '../../store/MockData';
+
+const ServerData = CreatedGames;
+const mockGame = {creator : "Anonymous", creatorColor: "",  opponent : "Anonymous", gameId : 2, gameType : "Classic", gameTime : 5, gameAddTime : 15, gameStarted : false, player1Ready: false, player2Ready: false,}; 
+
+
+interface IStateType  {
+  Player1Ready: boolean;
+  Player2Ready: boolean;
+  IsPlayer1: boolean;
+  gameId: number;
+ 
+}
+class LobbyAwaitGameComponent extends React.Component<any, IStateType> {
+
+  constructor (props: any) {
+    super(props)
+    this.state = {             
+      gameId: this.props.match.params.id,     
+      IsPlayer1: true,
+      Player1Ready: false,
+      Player2Ready: false,
+    };
+    
+}
+
+public SetReadyPlayer1 = (GameData : any) => {
```

Comment:
GameData should be gameData and should be typed up and not any. Example it contains all the information already.
public SetReadyPlayer1=  (gameData:  IGameData) =>
interface IGameData {
creator: string
creatorColor: string
gameAddTime: number
gameId: number
gameStarted: boolean
gameTime: number
gameType: 'Classic' | undefined
opponent: string
player1Ready: boolean
player2Ready: boolean
}

Reply:
Done